backends/edk2-compat/edk2-svc-validate.c

```diff
@@ -429,6 +460,17 @@ static int validateSingularESL(size_t *bytesRead, const unsigned char *esl, size
 			prlog(PR_INFO, "\tHash: ");
 			printHex(cert, cert_size);
 		}
+	}else if (varName && !strcmp(varName, "sbat")) {
+		if (!validate_sbat(cert)) {
+			prlog(PR_ERR,
+			      "ERROR: sbat data format is invalid\n");
+			rc = SBAT_FAIL;
+		} else
+			rc = SUCCESS;
+
+		if (verbose >= PR_INFO) {
+			prlog(PR_INFO, "\tSBAT: \n%s\n", cert);
```

Comment:
This is another cause of failed memcheck tests. To fix use `printRaw`.

Reply:
fixed it